bitnami/mariadb/templates/primary/statefulset.yaml

```diff
@@ -30,10 +32,14 @@ spec:
         {{- if (include "mariadb.primary.createConfigmap" .) }}
         checksum/configuration: {{ include (print $.Template.BasePath "/primary/configmap.yaml") . | sha256sum }}
         {{- end }}
+        {{- if .Values.passwordUpdateJob.enabled }}
+        charts.bitnami.com/password-last-update: {{ now | date "20060102150405" | quote }}
```

Comment:
I'm using charts.bitnami.com as prefix because, when we add this kind of annotations, we get several users asking if this is a standard annotation. This way we explicitly say that this comes from our own definitions, no Kubernetes standard.